deploy/warehouse/target/data.sql

```diff
@@ -5,7 +5,17 @@ begin;
 
 insert into warehouse.organism (lineage, identifiers, details)
     values
+        ('Adenovirus.1',            null,                       null),
+        ('Adenovirus.2',            null,                       null),
         ('Bordetella_pertussis',    'NCBITAXON => 520',         '{"report_to_public_health": true}'),
+        ('Chlamydophila_pneumoniae',null,                       null),
+        ('Coronavirus.229E_OC43',   null,                       null),
```

Comment:
Ugh, yeah. Looks like we'll need a `target_organism` link/join table. Since it'll require more work, for now I think we can punt on the cases like this where a target is linked to multiple specific species/subtypes. We can address it later if necessary.

Reply:
Ugh, yeah. Looks like we'll need a `target_organism` link/join table. Since it'll require more work, for now I think we can punt on the cases like this where a target is linked to multiple specific species/subtypes. We can address it later if necessary.

Reply:
Do you mean that I should ignore Coronavirus for now?

Reply:
Yep. If you would create a card in Trello so we remember this needs following up, that'd be great.